<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureResearchClass</class>
 <widget class="QMainWindow" name="PictureResearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureResearch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabBitmapFileHeader">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <attribute name="title">
         <string>位图头文件数据</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>251</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelBfType">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>文件类型</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelBfSize">
            <property name="text">
             <string>文件大小</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelBfReserved2">
            <property name="text">
             <string>保留值2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelBfReserved1">
            <property name="text">
             <string>保留值1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelBfOffBits">
            <property name="text">
             <string> 距图像数据
的偏移量</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEtBfType"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEtBfSize">
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEtBfReserved1"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEtBfReserved2"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEtBfOffBits"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabBitmapInfoHeader">
        <attribute name="title">
         <string>位图信息数据</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>431</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="labelBiWidth">
            <property name="text">
             <string>图像宽度</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelBiSize">
            <property name="text">
             <string>位图信息数据
大小</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelBiHeight">
            <property name="text">
             <string>图像高度</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelBiCompression">
            <property name="text">
             <string>压缩类型</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEtBiBitCnt"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEtBiWidth"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelBiPlane">
            <property name="text">
             <string>位面数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEtBiSize"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEtBiPlane"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEtBiHeight"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelBiBitCnt">
            <property name="text">
             <string>颜色位数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelBiSizImg">
            <property name="text">
             <string>图像实际大小</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelBiClrUsed">
            <property name="text">
             <string>颜色索引数</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelBiXPelsPerM">
            <property name="text">
             <string>水平分辨率</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEtBiCompression"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelBiYPelsPerM">
            <property name="text">
             <string>垂直分辨率</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="labelBiClrImportant">
            <property name="text">
             <string>重要颜色索引数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEtBiSizImg"/>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEtBiXPelsPerM"/>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEtBiYPelsPerM"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEtBiClrUsed"/>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEtBiClrImportant"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>487</width>
          <height>477</height>
         </rect>
        </property>
        <widget class="QLabel" name="LabPicture">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>441</width>
           <height>441</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="PictureResearch.qrc">:/images/img/spiderman.jpg</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="icon">
    <iconset>
     <normalon>:/images/img/open3.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PictureResearch.qrc"/>
 </resources>
 <connections/>
</ui>
